test/danetest.c

```diff
@@ -192,7 +190,7 @@ static ossl_ssize_t hexdecode(const char *in, void *result)
     for (byte = 0; *in; ++in) {
         int x;
 
-        if (isspace(_UC(*in)))
+        if (isspace(*in))
```

Comment:
The _UC() should be removed for ossl_isXXX calls. Also @bernd-edlinger indicates that _UC() is a compiler reserved string.